src/functions/pollJournals/framework/repo/mysql/pool.ts

```diff
@@ -8,6 +8,12 @@ export const createConnectionPool = (): mysql.Pool => {
     database: configuration.tarsReplicaDatabaseName,
     user: configuration.tarsReplicaDatabaseUsername,
     password: configuration.tarsReplicaDatabasePassword,
+    ssl: process.env.TESTING_MODE ? null : 'Amazon RDS',
+    authSwitchHandler(data, cb) {
```

Comment:
What's this for?

Reply:
This is the configuration that enables the IAM RDS Auth to work on the connection, as seen in the example here: https://cloudonaut.io/passwordless-database-authentication-for-aws-lambda/